crates/axvm/src/arch/aarch64/mod.rs

```diff
@@ -8,15 +10,13 @@ mod vcpu;
 use core::arch::asm;
 use spin::once::Once;
 
-use axhal::arch::register_exception_handler_aarch64;
+use axhal::arch::register_lower_aarch64_synchronous_handler;
```

Comment:
Pay attention to `exception_vector_base_el2` in your modifications to [arceos/modules/axhal](https://github.com/arceos-hypervisor/arceos/blob/4721e26b7bc823bfc7efcb07c82d90f5afdb9558/modules/axhal/src/arch/aarch64/hv/exception.S#L90C1-L90C26)

You only set `lower_xxx_handler`, while ignoring `current_xxx_handler` ?

Who is responsible for handling irq (e.g. time irq) in EL2?

Reply:
I add current irq handler in axhal just like el1. We should remove this after implementing interrupt virtualization in the future.
As for current synchronous exception handler, maybe letting it to panic is ok right now?

Reply:
* For the former, this is unrelated to interrupt virtualization. Even at the hypervisor privilege level,  there are still interrupts that belong to the hypervisor itself, such as hv's clock interrupts and certain driver interrupts (assuming hv has its own devices). Therefore, these still need to be preserved.

* For the latter, you are correct.